src/vm/analysis/type_checker/tests/assets.rs

```diff
@@ -100,31 +97,31 @@ fn test_bad_asset_usage() {
     use vm::analysis::type_check;
 
     let bad_scripts = ["(ft-get-balance stackoos tx-sender)",
-                       "(ft-get-balance 1234 tx-sender)",
-                       "(ft-get-balance stackaroos 100)",
-                       "(nft-get-owner 1234 \"abc\")",
+                       "(ft-get-balance u1234 tx-sender)",
```

Comment:
Can you please preserve the tests with signed integers as well?

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1129/commits/50149b7a553118e70688d3cd2ea47e9e516683c4